### ISSUE TITLE
::when should imply ::get

### ISSUE THREAD

**Issue by scravy** [scravy/jinsi#5]
`::when` is always used in conjunction with `::get`:

```
::when:
  ::get: ...
```

The content of `::when` should always be a `::get` such that

```
::when:
  ::get: a == 3
```

simply becomes

```
::when: a == 3
```

**Comment by GOusignu** [comment]
Hey! I would like to work on this

**Comment by scravy** [comment]
@GOusignu Welcome :-) You're probably off to a good start by looking at `parse_conditional` in `Parser`

**Comment by scravy** [comment]
Fixed by #8
